src/thunderbots/software/geom/util.h

```diff
@@ -469,6 +482,18 @@ bool pointInFrontVector(Point offset, Point dir, Point p);
 std::pair<Point, Point> getCircleTangentPoints(const Point &start, const Circle &circle,
                                                double buffer = 0.0);
 
+/**
+ * Returns the tangent vectors of a Circle and Point (Vectors are directed towards Circle)
+ *
+ * @param reference: The point which the tangent vectors will intersect
+ * @param circle: The circle to calculate the tangent vectors of
+ * @param buffer:
```

Comment:
`buffer` needs a comment

Reply:
True. Buffer is used in the getCircleTangentPoints() that I use in this function. I've decided to remove buffer as a parameter for this getCircleTangentRays() (renamed)